vector/src/main/java/im/vector/app/features/home/room/detail/timeline/item/MessageVoiceItem.kt

```diff
@@ -93,33 +120,41 @@ abstract class MessageVoiceItem : AbsMessageItem<MessageVoiceItem.Holder>() {
         holder.voicePlaybackLayout.backgroundTintList = ColorStateList.valueOf(backgroundTint)
         holder.voicePlaybackControlButton.setOnClickListener { playbackControlButtonClickListener?.invoke(it) }
 
-        voiceMessagePlaybackTracker.track(attributes.informationData.eventId, object : VoiceMessagePlaybackTracker.Listener {
-            override fun onUpdate(state: VoiceMessagePlaybackTracker.Listener.State) {
-                when (state) {
-                    is VoiceMessagePlaybackTracker.Listener.State.Idle    -> renderIdleState(holder)
-                    is VoiceMessagePlaybackTracker.Listener.State.Playing -> renderPlayingState(holder, state)
-                    is VoiceMessagePlaybackTracker.Listener.State.Paused  -> renderPausedState(holder, state)
+        // Don't track and don't try to update UI before view is present
```

Comment:
is this because we're relying on [`holder.voicePlaybackWaveform.post`](https://github.com/vector-im/element-android/pull/5404/files#diff-986f29a475d699ca21a7504d49eba50b6567a25b3ea9284b82bc17bd854db924R93)?  if so we should move the tracker logic to the same `post` body to better couple the logic together (it seems that we're relying on the audio view to be fully drawn because it only updates its internal state as a side effect of the drawing)

Reply:
Yes we need `post` because the width of the view is 0 (not calculated yet) when it is bind.